endpoints.json

```diff
@@ -42,5 +42,27 @@
         "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
       }
     }
+  },
+  "GET /api/articles/:article_id/comments": {
+    "description": "serves an array of all comments for the specified article",
+    "queries": [],
+    "exampleResponse": [
```

Comment:
Don't forget to include the key you return the data on. You'll thank yourself later!